docs/observations.rst

```diff
@@ -0,0 +1,9 @@
+Observations
+==========
```

Comment:
Needs to be longer (same number of equals signs as characters in `Observations`).
```
reading sources... [100%] tier1a                                                
/home/xylar/code/mpas-work/analysis/adds_observational_xml/docs/index.rst:34: WARNING: toctree contains reference to nonexisting document 'mpascice'
/home/xylar/code/mpas-work/analysis/adds_observational_xml/docs/observations.rst:2: WARNING: Title underline too short.

Observations
==========
```